src/App.jsx

```diff
@@ -1,104 +1,24 @@
 import './App.scss';
 
-// import postsFromServer from './api/posts.json';
-// import commentsFromServer from './api/comments.json';
-// import usersFromServer from './api/users.json';
+import postsFromServer from './api/posts.json';
+import commentsFromServer from './api/comments.json';
+import usersFromServer from './api/users.json';
+import { PostList } from './components/PostList/PostList';
+
+const posts = postsFromServer.map(post => {
+  const user = usersFromServer.find(item => item.id === post.userId);
+  const comments = commentsFromServer.filter(item => item.postId === post.id);
+
+  return {
+    ...post,
+    user,
+    comments,
+  };
+});
```

Comment:
You can write a clear function for it then import and use it in the component for preparing posts ( for the future)